### ISSUE TITLE
Loading script from dat url fails

### ISSUE THREAD

**Issue by paul90** [sammacbeth/dat-fox#24]
In a `index.html` scripts loaded from a dat url fail to load. For example

```
<script src='dat://federated-wiki-client.hashbase.io/client.js' type='text/javascript'></script>
``` 
fails to load with `Loading failed for the <script> with source “dat://federated-wiki-client.hashbase.io/client.js”.` in the browser console.

**Comment by paul90** [comment]
Discovered while looking at using Content Security Policy to resolve the the CORS issue raised in https://github.com/sammacbeth/dat-fox/issues/23

**Comment by paul90** [comment]
The code in content_script.js  
https://github.com/sammacbeth/dat-fox/blob/db4f5976b7eb2b03bae8f72c5fd5a7ca44130896/addon/content_script.js#L32-L39
does not appear to being called. If I run it in the console it works, and the scripts are loaded. 

This is with Firefox Dev. Edition 63.0b13 on macOS.

**Comment by raphaelbastide** [comment]
Is there some news regarding this issue and the experimental version with the new protocol?

**Comment by sammacbeth** [comment]
> Is there some news regarding this issue and the experimental version with the new protocol?

Yes. It's [dat-webext](https://github.com/cliqz-oss/dat-webext) which bundles the whole dat stack inside the extension. Details on how it works are in this [blog post](https://sammacbeth.eu/blog/2019/05/12/dat-for-firefox-2.html).

Unfortunately, as it uses experimental APIs, it cannot be easily shipped to Firefox users at present, though Mozilla are looking into featuring dweb extensions like this on the AMO store, and permitting their use of experimental APIs. In the meantime, it can be tested on Desktop in [Cliqz's Beta](https://cliqz.com/en/latest) channel (a Firefox fork), or on Android in [Cliqz Concept Browser](https://github.com/cliqz/cliqz-concept-browser).

**Comment by raphaelbastide** [comment]
@sammacbeth Thank you very much for that article and your work.
